src/sheets/quadrone/item/parts/ItemStartingEquipment.svelte

```diff
@@ -40,9 +40,7 @@
     </div>
   </div>
   <div class="form-group">
-    <label for="{appId}-starting-equipment"
-      ></label
-    >
+    <label for="{appId}-starting-equipment" class="{!context.unlocked ? 'no-hover' : ''}"></label>
```

Comment:
We can just remove the `for` attribute to prevent hover styles. This is normal HTML behavior.